src/modules/domains/domains.service.ts

```diff
@@ -78,28 +87,31 @@ export class DomainsService {
       ensRegistryAddress,
       provider,
     });
-    ensPublicResolverAddress &&
-      this._domainDefinitionReader.addKnownResolver({
-        chainId,
-        address: ensPublicResolverAddress,
-        type: ResolverContractType.PublicResolver,
-      });
```

Comment:
@JGiter Thanks for making the changes. I think we need to support the `PublicResolver` though until we're pretty sure that we've migrated all of the old domains though, right? Then we can remove the `PublicResolver` code?

Reply:
Ok